Automattic/wp-calypso
# Free Onboarding Flow: Update post setup screen to match initial setup screen & update test cases

#### Proposed Changes

* Updated Free Flow Post setup screen text to match the initial setup screen
* Removed "Free" flow title from the header
* Added test cases

#### Testing Instructions



* Checkout this branch
* Create a new site or use an existing site slug.
* Confirm the Free flow post setup screen test cases are passing: `yarn run test-client client/landing/stepper/declarative-flow/internals/steps-repository/free-post-setup/test/index.tsx`
* Compare the free flow initial setup screen (`/setup/free/freeSetup`) and post setup screen (`/setup/free-post-setup/freePostSetup?siteSlug={siteSlug}`)

Initial free flow setup screen:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/10482592/210898197-8529c0a5-04f0-4b9c-be8e-9e35aa6cc2b9.png">

Post setup screen:
<img width="1507" alt="image" src="https://user-images.githubusercontent.com/10482592/210898421-1b8ea18a-70ba-4d93-a215-0fc99b2be76d.png">





#### Pre-merge Checklist





Related to #71354
